cmd/juju/config/config.go

```diff
@@ -146,6 +142,10 @@ func (c *ConfigCommandBase) parseResetKeys() error {
 
 // sliceContains is a utility method to check if the given (string) slice
 // contains a given value.
+// NB: The slices package
+// https://pkg.go.dev/golang.org/x/exp/slices
+// will be released in a future version of Go - once that happens, we should
+// remove this method and use slices.Contains instead.
```

Comment:
You can probably leave this comment out -- there's no doubt tons of places in Juju where this applies, so when we upgrade to Go 1.19 in the future we can go around and clean them all up.